runtime-sdk-typescript/index.ts

```diff
@@ -145,13 +145,25 @@ const main = async () => {
     account,
     { type: "local", input: { headless: false } },
   );
-  executeRuntimeScript({
+  await executeRuntimeScript({
     client,
     account,
     target: { browserContext: browserContext },
     initialUrl: appUrls[account.app],
     run: actions[account.app],
   });
+  const accountInfo = { ownerId: APP_USER_ID, domain: account.app };
+
+  // DEMONSTRATION of `getSessionStatus`, `deleteSession`
+  let sessionStatus;
+  sessionStatus = await client.getSessionStatus({ account: accountInfo });
```

Comment:
A bit cleaner to just do 
```suggestion
  let sessionStatus = await client.getSessionStatus({ account: accountInfo });
```
Unbound variables are not desired